src/components/GameView/GameMenu.vue

```diff
@@ -8,19 +8,22 @@
         </v-btn>
       </template>
       <!-- Menu -->
-      <v-list id="game-menu">
-        <v-list-item data-cy="rules-open" @click.stop="showRulesDialog = true">
+      <v-list id="game-menu" class="bg-surface-2">
+        <v-list-item data-cy="rules-open" :size="buttonSize" @click.stop="openRulesDialog">
```

Comment:
Does this actually need the modifier `.stop`? Not sure if there are other elements that would fire a click action passed this point.

Reply:
Yes you are right, i am removing it, thanks for the feedback !